Server/config/default.json

```diff
@@ -11,5 +11,8 @@
         "cluster3": "cluster0-shard-00-00-jt05i.mongodb.net",
         "port": 27017,
         "query": "ssl=true&replicaSet=Cluster0-shard-0&authSource=admin&retryWrites=true&w=majority"
+    },
+    "passport" : {
+        "secret" : "secretkey"
```

Comment:
Please change value to something line `tms_jwt_secret`

Reply:
Updated with tms_jwt_secret